src/components/DpSearchField/DpSearchField.vue

```diff
@@ -4,16 +4,19 @@
       id="searchField"
       data-cy="searchField"
       :class="cssClasses"
-      :input-attributes="{ placeholder: Translator.trans('search'), type: 'search' }"
+      :input-attributes="{ placeholder: Translator.trans('search'), type: 'search', inputClasses: 'rounded-r-none' }"
       @reset="handleReset"
       @enter="handleSearch"
       v-model="searchTerm" /><!--
 
  --><dp-button
-      class="align-top"
-      data-cy="handleSearch"
-      @click="handleSearch"
-      :text="Translator.trans('searching')" />
+    class="align-top search rounded-r-md border-l-0 rounded-l-none"
```

Comment:
- why is `align-top` even necessary?
- i guess with `border-l-0` the left border remains grey. does this win over a blue border visually?
- what does `search` do here? i could not spot the class anywhere
- Nitpick: indentation

Reply:
> * i guess with `border-l-0` the left border remains grey. does this win over a blue border visually?

Fixed here https://github.com/demos-europe/demosplan-core/pull/4193/commits/c1cf03c9335f65591e3a368aed4d0e6e9154712e and here https://github.com/demos-europe/demosplan-ui/pull/1133/commits/1887d90afd61fc3301ba3ff6d9ff2722e0036723